api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -437,12 +438,22 @@ public void testGetRuntime_error() throws ApiException {
   }
 
   @Test
-  public void testGetRuntime_errorNoMessages() throws ApiException {
+  public void testGetRuntime_error_nullMessages() throws ApiException {
     when(mockUserRuntimesApi.getRuntime(GOOGLE_PROJECT_ID, getRuntimeName()))
         .thenReturn(testLeoRuntime.status(LeonardoRuntimeStatus.ERROR).errors(null));
 
     assertThat(runtimeController.getRuntime(WORKSPACE_NS).getBody())
-        .isEqualTo(testRuntime.status(RuntimeStatus.ERROR));
```

Comment:
this was no longer true, because the testRuntime has an empty list of errors, not null.  (yes I just made the change that broke this test, but it was necessary to rescue exact matches in other tests)

I split this test into the 2 cases.